MeesGame/UI/Controls/Button.cs

```diff
@@ -23,7 +24,7 @@ public class Button : UIComponent
         /// The textbox inside the Button.
         /// </summary>
         private Textbox textBox;
-
+        
```

Comment:
I don't really know what would be wrong with the formatting, but since I didn't work on it (Dennis did), I might not see the problem